code/modules/hydroponics/grown/nymph.dm

```diff
@@ -11,6 +11,44 @@
 	production = 1
 	yield = 1
 	reagents_add = list("plantmatter" = 0.1)
+	var/next_ping_at = 0
+
+// If the nymph pods are fully grown, allow ghosts that click on them to spawn as a nymph (decreases yield by 1 and kills the plant when yield reaches 0)
+/obj/item/seeds/nymph/attack_ghost(mob/dead/observer/O, obj/machinery/hydroponics/H)
+	if(!planted && !H)
+		if(isobserver(O) && (world.time >= next_ping_at))
+			next_ping_at = world.time + (20 SECONDS)
+			visible_message("<span class='notice'>[src] rustles gently, as if moved by a gentle breeze.</span>")
+		return
+	if(!H.harvest) 
+		to_chat(O, "[src] is not yet ready.")
```

Comment:
This needs `<span>`'s around the message text, likewise for the other `to_chat`s in this proc.